libs/langchain/langchain/chains/moderation.py

```diff
@@ -33,6 +38,7 @@ class OpenAIModerationChain(Chain):
     output_key: str = "output"  #: :meta private:
     openai_api_key: Optional[str] = None
     openai_organization: Optional[str] = None
+    openai_version: Optional[str] = None
```

Comment:
if you want you can also do this, and then you don't need the mypy stuff below (validation guarantees it's populated)
```suggestion
    openai_version: str = Field(default=None)
```